Source/Scene/Scene.js

```diff
@@ -1344,6 +1396,8 @@ define([
 
         if (scene.debugShowCommands || scene.debugShowFrustums) {
             executeDebugCommand(command, scene, passState);
+        } else if (scene.frameState.shadowMaps.length > 0 && command.receiveShadows && defined(command.derivedCommands.shadows.receiveCommand)) {
```

Comment:
Is the last check, `defined(command.derivedCommands.shadows.receiveCommand)`, needed?